src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/PlacementRequestEntity.kt

```diff
@@ -61,10 +61,11 @@ interface PlacementRequestRepository : JpaRepository<PlacementRequestEntity, UUI
         END
       ) = :#{#status?.toString()})
       AND (:crn IS NULL OR (SELECT COUNT(1) FROM applications a WHERE a.id = pq.application_id AND a.crn = UPPER(:crn)) = 1)
+      AND (:tier IS NULL OR (SELECT COUNT(1) FROM approved_premises_applications apa WHERE apa.id = pq.application_id AND apa.risk_ratings -> 'tier' -> 'value' ->> 'level' = :tier) = 1)
```

Comment:
Unless I'm missing something, there doesn't appear to be any tests for the tier search? Would be good to get something in, especially as we're querying over JSON here.

Reply:
Good spot, have added a test for this